JapaneseFilter/sections/css_extended.txt

```diff
@@ -103,6 +103,7 @@ otonasalone.jp,sumaiweb.jp#?#._popIn_infinite_page > a._popIn_recommend_article_
 !
 zakzak.co.jp#?#.sidebar-items > header.story-card-header.large:contains(zakスペシャル)
 zakzak.co.jp#?#.p-10px > div.OUTBRAIN:upward(1)
+techable.jp#?#.te-article__body__inner > div.te-article__section:has(> script[src^="https://speee-ad.akamaized.net"])
```

Comment:
This sort of rule doesn't work on apps as `script[src^="https://speee-ad.akamaized.net"]` will be removed before the rule is applied. Use `.uz-ny` instead.